src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/datastore/fiks/SoknadService.kt

```diff
@@ -244,20 +247,37 @@ class SoknadService {
         fillager.add(vedleggsId, filnavn, bytes)
         return vedleggsId
     }
+
+    fun hentSoknadstittel(fiksDigisosId: String): String {
+        val digisosSak = soknadsliste[fiksDigisosId]
+        val soknadString = dokumentLager[digisosSak!!.digisosSoker!!.metadata]
+        val soknad = objectMapper.readValue(soknadString, JsonDigisosSoker::class.java)
+        val saksTittelMap = HashMap<String, String>()
+        soknad.hendelser.filter { it.type == JsonHendelse.Type.SAKS_STATUS }
+                .forEach {
+                    if (it is JsonSaksStatus) {
+                        saksTittelMap.put(it.referanse, it.tittel)
+                    }
+                }
```

Comment:
Kan dette kanskje forenkles noe (ala dtte utkastet)?
```
soknad.hendelser
    .filter { it.type == JsonHendelse.Type.SAKS_STATUS && it is JsonSaksStatus}
    .map { it.referanse to it.tittel }
    .toMap()
```

Reply:
Ja, jeg hadde helt glemt denne map { a to b } tingen. Jeg skriver det bak øret.